test/Passes/test-matchers.mlir

```diff
@@ -250,3 +250,63 @@ func.func @test_tpp_identity(%arg0: memref<3xf32>, %arg1: memref<5x3xf32>) {
   } 
   return
 }
+
+#map13 = affine_map<(d0, d1) -> (0)>
+#map14 = affine_map<(d0, d1) -> (d0, d1)>
+#map15 = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
+#map16 = affine_map<(d0, d1) -> ()>
+
+// CHECK-LABEL: test_rank
+func.func @test_rank(%arg0: memref<1xf32>, 
+                     %arg1: memref<4x4xf32>, %arg2: memref<4x4x4xf32>, 
+                     %arg3: f32) {
+  // CHECK: match 0
```

Comment:
These match output makes no sense. Please always abide to the "principle of least surprise".